main.js

```diff
@@ -0,0 +1,525 @@
+                                        box.forEach(ele => {
+                                            ele.style.pointerEvents = "none";
+                                        })
+                                    }
+                                } else if (canLose(position)) {
+                                    moveElement(canLose(position).idx, position);
+                                    if (playerTwo.checkVictory(position)) {
+                                        let child = document.createElement("h1");
+                                        child.classList.add("winner");
+                                        child.innerHTML = `${playerTwo.winner()} `
+                                        info.appendChild(child);
+                                        box.forEach(ele => {
+                                            ele.style.pointerEvents = "none";
+                                        })
+                                    }
+                                } else {
+                                    movePc(position)
+                                }
+                            } else {
+                                movePc(position)
+                            }
+                        }, 1000)
+                    }
+                }
+            })
+        })
+    }
+    //function for generating computer move-------------------------------------------
+    function movePc(position) {
+        let validCell = position.filter(ele => ele !== "O" && ele !== "X")
+        let randomChoice = Math.floor(Math.random() * validCell.length);
+        idx = position.indexOf(validCell[randomChoice]);
+        box.forEach(ele => {
+            if (ele.getAttribute("id") === validCell[randomChoice]) {
+                ele.textContent = playerTwo.symbol;
+                position[idx] = playerTwo.symbol;
+                ele.setAttribute("id", idx);
+            }
+        })
+        turnBox.textContent = playerOne.name;
+        if (playerTwo.checkVictory(position)) {
+            let child = document.createElement("h1");
+            child.classList.add("winner");
+            child.innerHTML = `${playerTwo.winner()}`;
+            info.appendChild(child);
+            box.forEach(ele => {
+                ele.style.pointerEvents = "none";
+            })
+        }
+    }
+    //creating game timer
+    let minutesLabel = document.getElementById("minutes");
+    let secondsLabel = document.getElementById("seconds");
+    let totalSeconds = 0;
+    setInterval(setTime, 1000);
+
+    function setTime() {
+        ++totalSeconds;
+        secondsLabel.innerHTML = pad(totalSeconds % 60);
+        minutesLabel.innerHTML = pad(parseInt(totalSeconds / 60));
+    }
+
+    function pad(val) {
+        let valString = val + "";
+        if (valString.length < 2) {
+            return "0" + valString;
+        } else {
+            return valString;
+        }
+    }
+
+
+    function canWin(position) {
```

Comment:
This function has a lot of repetitive code which is a sign that it could probably be abstracted a bit and refactored to a much shorter function. This is a good place to start now look and see what code is the same between these if statements, and what might need to be passed in as a variable.